### PR TITLE
New GPU InstanceTypes for Provisioning

### PR DESCRIPTION
**Pull request by SrinivasAtmakuri**
JIRA: https://issues.redhat.com/browse/ARO-4606

Adds below InstanceTypes for provisioning during
cluster creation.

### Which issue this PR addresses:

Fixes

### Test plan for issue:

1. Create clusters using the new InstanceTypes

### CODE REVIEW

File: pkg/api/openshiftcluster.go [hunk 1]
```diff
@@ -598,31 +604,39 @@ var (
...
+	standardEIDSv5      = "standardEIDSv5Family"
+	standardEIDv5       = "standardEIDv5Family"
+	standardFSv2        = "standardFSv2Family"
+	standardMS          = "standardMSFamily"
+	standardLSv2        = "standardLsv2Family"
+	standardLSv3        = "standardLsv3Family"
+	standardNCAS        = "Standard NCASv3_T4 Family"
+	standardNCSv3       = "Standard NCSv3 Family"
+	standardNCADSA100v4 = "StandardNCADSA100v4Family"
+	standardNDASA100v4  = "Standard NDASv4_A100 Family"
```
Comment (cadenmarchese) [inline pkg/api/openshiftcluster.go:639]: There's some inconsistency in this file, can we correct it while we're here?
```suggestion
	standardNCAS        = "standardNCASv3T4Family"
	standardNCSv3       = "standardNCSv3Family"
	standardNCADSA100v4 = "standardNCADSA100v4Family"
	standardNDASA100v4  = "standardNDASA100v4Family"
```
etc..

Comment (SrinivasAtmakuri) [reply pkg/api/openshiftcluster.go]: Made the changes. Thanks.

**Review by cadenmarchese — APPROVED** [review]

**Comment by SrinivasAtmakuri** [comment]
This PR is on hold, more details on the JIRA are here: https://issues.redhat.com/browse/ARO-3428?filter=-1

**Comment by SrinivasAtmakuri** [comment]
// Closing, as the issue is blocked

**Pull request closed without merging**